frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx

```diff
@@ -293,6 +316,49 @@ function Pipelines({ type }) {
     },
     {
       key: "2",
+      label: (
+        <Space
+          direction="horizontal"
+          className="action-items"
+          onClick={() =>
+            getApiKeys(
+              pipelineApiService,
+              selectedPorD?.id,
+              setApiKeys,
+              setOpenManageKeysModal
+            )
+          }
+        >
+          <div>
+            <KeyOutlined />
+          </div>
+          <div>
+            <Typography.Text>Manage Keys</Typography.Text>
+          </div>
+        </Space>
+      ),
+    },
+    {
+      key: "3",
+      label: (
+        <Space
+          direction="horizontal"
+          className="action-items"
+          onClick={() =>
+            downloadPostmanCollection(pipelineApiService, selectedPorD?.id)
+          }
+        >
+          <div>
+            <CloudDownloadOutlined />
+          </div>
+          <div>
+            <Typography.Text>Download Postman Collection</Typography.Text>
+          </div>
```

Comment:
Do we need these div here? Can't we get the same result but adding styles to space component

Reply:
@jagadeeswaran-zipstack Good catch. I had refered this part of the code from `/deployments/api-deployment/ApiDeployment.jsx` component. So, I missed removing the wanted div tags. I have made the change now.